src/components/pages/HomeView/JoinWaitlist.tsx

```diff
@@ -0,0 +1,50 @@
+import { useIntl } from "react-intl";
+import { motion } from "framer-motion";
+
+const JoinWaitlist = () => {
+  const i18n = useIntl();
+
+  const joinTheVeridaWaitlistTitle = i18n.formatMessage({
+    id: "HomeView.JoinWaitlistTitle",
+    description: "",
+    defaultMessage: "Join the Verida One waitlist",
+  });
+
+  const joinTheWaitlistButtonLabel = i18n.formatMessage({
+    id: "HomeView.JoinTheWaitlistButtonLabel",
+    description: "",
+    defaultMessage: "Join the waitlist",
+  });
+
+  return (
+    <motion.div
+      initial="hidden"
+      whileInView="visible"
+      viewport={{ amount: 0.3, once: true }}
+      variants={{
+        hidden: {
+          y: 80,
+          opacity: 0,
+        },
+        visible: {
+          y: 0,
+          opacity: 100,
+        },
+      }}
+      transition={{ duration: 0.8 }}
+      className="!mt-40 flex flex-col"
```

Comment:
This margin should not be applied by the component root tag as it's not related to the style/layout of the component itself. It sshould be applied by whatever component is rendering it (HomeView in that case). The HomeView should either surround this component with a div and add the margin there. Or the props of this JoinWaitlist component should take the style and merge it with there in the root tag, so that HomeView apply the style directly on the component. (Even better, the props of JoinWaitlist should extend `ComponentProps<'div'>` and be passed to the root div tag, that's actually what I tend to do personally but I notice I haven't really done it in this repo)

Reply:
Resolved